### PR TITLE
add changes that set up a express server, mysql database, and a webPack

### PR DESCRIPTION
**Pull request by jpkanderton**
work in progrss

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE nearbyHomes;
```
Comment (dsl8va) [inline schema.sql:1]: you could also add IF NOT EXISTS it only creates the database or table if they don't exist

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const port = 3000;
+const bodyParser = require('body-parser');
+// const database = require('./database/database.js');
+
+
+app.use(express.static(__dirname + '/../dist'));
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+app.get('/api/homes', (req, res) => {
```
Comment (dsl8va) [inline server/server.js:12]: you could split up the server side into models and controllers where the controllers handle the get/post requests and the models handle querying the database. not required, but similar to what we've seen in some past sprints.

**Review by dsl8va — COMMENTED** [review]
Just added two comments. One about the sql file, the other about your api handling in the server.

File: schema.sql (hunk 1 above)
Comment (jpkanderton) [reply schema.sql:1]: thanks! i'll add this in

File: server/server.js (hunk 2 above)
Comment (jpkanderton) [reply server/server.js:12]: i'll implement these changes as well--better to have a more professionally organized repository

**Pull request merged**